### PR TITLE
fix: make code name editable for migration table (contract detail page)

### PR DESCRIPTION
**Pull request by bkioshn**
## Describe your changes
- Make code editable for migration table in contract detail page
- Get uploader info

### CODE REVIEW

File: src/lib/pages/contract-details/components/tables/migration/MigrationRow.tsx [hunk 1]
```diff
@@ -68,7 +70,15 @@ export const MigrationRow = ({
...
       </TableRow>
       <TableRow>
-        {history.codeName || <Text color="text.dark">Untitled Name</Text>}
+        <CodeNameCell
+          code={
+            {
+              id: history.codeId,
+              uploader: history.uploader,
+              name: history.codeName,
+            } as CodeLocalInfo
```
Comment (evilpeach) [inline src/lib/pages/contract-details/components/tables/migration/MigrationRow.tsx:79]: you can remove assertion

File: src/lib/components/table/EditableCell.tsx [hunk 2]
```diff
@@ -90,6 +90,8 @@ export const EditableCell = ({
     isShowInputValue &&
     Number(textRef.current?.scrollWidth) > Number(textRef.current?.clientWidth);
 
+  useEffect(() => setInputValue(initialValue), [initialValue]);
```
Comment (evilpeach) [inline src/lib/components/table/EditableCell.tsx:93]: @songwongtp Last time I remembered, we just remove this for fixing some bug, right?

**Review by evilpeach — CHANGES_REQUESTED** [review]
Also update the branch.

File: src/lib/components/table/EditableCell.tsx (hunk 2 above)
Comment (songwongtp) [reply src/lib/components/table/EditableCell.tsx:93]: probably a different component

**Review by evilpeach — APPROVED** [review]

**Pull request merged**